sdk_extensions/aws_v1_support/src/test/java/io/opentelemetry/sdk/extensions/trace/aws/resource/DockerHelperTest.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.sdk.extensions.trace.aws.resource;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.common.base.Charsets;
+import com.google.common.io.Files;
+import java.io.File;
+import java.io.IOException;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+public class DockerHelperTest {
+
+  @Rule public TemporaryFolder tempFolder = new TemporaryFolder();
+
+  @Test
+  public void testCgroupFileMissing() {
+    DockerHelper dockerHelper = new DockerHelper("a_file_never_existing");
+    assertThat(dockerHelper.getContainerId()).isEmpty();
+  }
+
+  @Test
+  public void testContainerIdMissing() throws IOException {
+    File file = tempFolder.newFile("no_container_id");
+    String content = "13:pids:/\n" + "12:hugetlb:/\n" + "11:net_prio:/";
+    Files.write(content.getBytes(Charsets.UTF_8), file);
```

Comment:
Nit, a bit simpler as `Files.write(content, file, Charsets.UTF_8)`

Reply:
`Files.write(CharSequence, File, Charset)` is marked as deprecated, now the only available write method in class Files is `write(byte[], File)`

Reply:
Ah didn't know that, thanks! Wish the latest javadocs had enough SEO to show up on search